src/win/winsock.c

```diff
@@ -87,6 +87,9 @@ void uv_winsock_init(void) {
   WSAPROTOCOL_INFOW protocol_info;
   int opt_len;
 
+  /* Skip initialization in safe mode without network support */
+  if (1 == GetSystemMetrics(SM_CLEANBOOT)) return;
```

Comment:
This skips initializing `uv_addr_ip4_any_` and `uv_addr_ip6_any_`, which is probably a bad idea even in the absence of networking.

You can fix that by moving the block at lines 99-106 to before the GetSystemMetrics() call.